modules/puppet/files/com.mozilla.periodic_puppet.plist

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+
+    <!-- This Source Code Form is subject to the terms of the Mozilla Public
+       - License, v. 2.0. If a copy of the MPL was not distributed with this
+       - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+                <key>LANG</key>
+                <string>en_US.UTF-8</string>
+                <key>PATH</key>
+                <string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/puppetlabs/bin</string>
+        </dict>
+        <key>Label</key>
+        <string>com.mozilla.periodic_puppet</string>
+        <key>StartInterval</key>
+        <integer>900</integer>
+        <key>RunAtLoad</key>
+        <false/>
```

Comment:
I'm thinking when we first load this, a puppet run is already happening, so let's not kick off another one.